cypress/support/commands.js

```diff
@@ -10,6 +10,7 @@
 import '../../lib/pkp/cypress/support/commands';
 
 Cypress.Commands.add('addToCatalog', function() {
+	cy.get('#publication-button').click();
```

Comment:
Could you put this line into the test -- all the other tests are using it -- so maybe not to change this here now...

Reply:
Changed